project/AppDependencies.scala

```diff
@@ -1,39 +1,54 @@
+import play.sbt.PlayImport._
 import sbt._
 
 object AppDependencies {
-  import play.sbt.PlayImport._
 
   private val playVersion = "play-28"
 
-  private val pegdownVersion = "1.6.0"
-
   private val hmrcMongoVersion = "0.74.0"
```

Comment:
raise a ticket to update this to 1.x

Reply:
Created https://jira.tools.tax.service.gov.uk/browse/DDCNL-7702 to track this.